src/compress.ts

```diff
@@ -71,6 +71,25 @@ function isTextMime(mimeType: string) {
   return /text|javascript|json|xml/.test(mimeType);
 }
 
+  const compressibleMimeTypes = [
+    'atom',
+    'css',
+    'eot',
+    'htc',
+    'html',
+    'ico',
+    'js',
+    'json',
+    'mjs',
+    'otf',
+    'rss',
+    'svg',
+    'text',
+    'ttf',
+    'webmanifest',
+    'xml',
+  ];
+
 function isCompressableMime(mimeType: string) {
-  return /image|text|font|json|xml|javascript/.test(mimeType);
+  return compressibleMimeTypes.includes(mimeType);
```

Comment:
I'm not sure this will work as written, because aren't we testing a longer string like `text/xml`? Maybe something like:

```suggestion
  return compressibleMimeTypes.some(type => mimeType.includes(type));
```

Reply:
We could keep with the regex approach that was previously used and do something like:
```
const compressibleMimeTypesRegex = compressibleMimeTypes.join('|');
  return new RegExp(compressibleMimeTypesRegex).test(mimeType);
```

Reply:
Good idea. But let's use a regexp where we define it directly then to safe the overhead of converting a string => regexp.

Reply:
So... something like this should work:

```
const compressibleMimeTypesRegex = /atom|css|eot|htc|html|ico|js|json|mjs|otf|rss|svg|text|ttf|webmanifest|xml/

function isCompressible(mimeType) {
  return compressibleMimeTypesRegex.test(mimeType);
}
```

Reply:
Perfect.